### PR TITLE
C hbranch

### PR DESCRIPTION
**Pull request by NiarH**

### CODE REVIEW

File: source/guide/3.1/installation-general.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ The recommended requirements can vary based on the following:
 * Amount of concurrent logs and events you're going to send from your [hosts]({{page.terminology_link}}#host).
 * Amount of concurrent metrics you're going to send from your hosts.
 
-As a general recommendation for the average system, Cloudify would require at least 4GB of RAM and 4 CPU Cores. Disk space requirements varies according to the amount of logs, events and metrics sent as Cloudify doesn't currently clean them.
+As a general recommendation for the average system, Cloudify would require at least 4GB of RAM and 4 CPU Cores. Disk space requirement varies according to the amount of logs, events and metrics sent as Cloudify doesn't currently clean them.
```
Comment (nir0s) [inline source/guide/3.1/installation-general.md]: I think it should be "Disk space requirements vary..."

File: source/guide/3.1/installation-general.md [hunk 2]
```diff
@@ -60,13 +60,13 @@ Cloudify's management server currently runs on Ubuntu 12.04 Precise. To install
 Cloudify's Docker implementation was tested on Ubuntu 14.04 and Centos 6.5 and is based on the [phusion/baseimage](https://github.com/phusion/baseimage-docker) Docker Image (Ubuntu 14.04).
 
 {%note title=Note%}
-If the host machine Docker is running on is based on Ubuntu 14.04, we will attempt to install Docker for you (if it isn't already installed. Requires an internet connection). For any other distribution (and release), you'll have to verify that Docker is installed prior to bootstrapping.
+If the host machine Docker is running on is based on Ubuntu 14.04, we will attempt to install Docker for you, if it hasn't already been installed.(Requires an internet connection). For any other distribution (and release), you'll have to verify that Docker is installed prior to bootstrapping.
```
Comment (nir0s) [inline source/guide/3.1/installation-general.md]: Um... "hasn't already been installed..." is not grammatically possible :)
It should probably be "wasn't previously installed..."

File: source/guide/3.1/installation-manual.md [hunk 3]
```diff
@@ -46,3 +46,4 @@ For more information on the packages, please refer to the [Cloudify installation
 You can now direct your browser at http://manager-ip-address to access the UI or run `cfy use manager-ip-address` (depending on what interface you're using).
 
 Enjoy!
+uju
```
Comment (nir0s) [inline source/guide/3.1/installation-manual.md]: uju?

File: source/guide/3.1/quickstart.md [hunk 4]
```diff
@@ -269,6 +269,6 @@ vagrant box remove cloudify-box
 
 # What's Next
 
-* Understand the requirements for bootstrapping your very own Cloudify Manager by reading [this]({{page.installation_general_link}}).
+* Understand the requirements for bootstrapping your very own Cloudify Manager by reading under [Installation]({{page.installation_general_link}}).
```
Comment (nir0s) [inline source/guide/3.1/quickstart.md]: Reading under?

File: source/guide/3.1/installation-general.md (hunk 1 above)
Comment (NiarH) [inline source/guide/3.1/installation-general.md]: Yeah I thought so too thx

**Pull request closed without merging**